src/Sulu/Bundle/ContactBundle/Contact/ContactManagerInterface.php

```diff
@@ -40,21 +43,21 @@ public function removeAddressRelation($entity, $address);
     /**
      * Returns a collection of relations to get addresses.
      *
-     * @param string $entity
+     * @param ContactInterface $entity
```

Comment:
Same as above this can be an `AccountInterface` in case of AccountManager. we should use `object` or generics for this. This is also true for all other `ContactInterface` typehints in this Interface.

Reply:
For simple I would go here with `ContactInterface|AccountInterface`